core/Db/Schema/Mysql.php

```diff
@@ -522,10 +520,10 @@ public function createDatabase($dbName = null)
             $dbName = $this->getDbName();
         }
 
+        $createOptions = $this->getDatabaseCreateOptions();
```

Comment:
Just a minor suggestion — we're using `$tableOptions` var name above meaning table create options, here we use `$createOptions` var as db create options, might be good for consistency to always have the area it relates to (db/table) in the name in both places or in neither.

```suggestion
        $dbOptions = $this->getDatabaseCreateOptions();
```
or
```suggestion
        $dbCreateOptions = $this->getDatabaseCreateOptions();
```

Reply:
Never mind, I thought it was in the same method, but it's different methods, so no all good.